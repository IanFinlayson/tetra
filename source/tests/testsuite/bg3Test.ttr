def main() int:
   v = [0]
   doStuff(v)
   return v[0]

def doStuff(v [int]):
   x = 2
   background:
      for z in [1 ... 1000]:
         pass
      v[0] += x
   pass
