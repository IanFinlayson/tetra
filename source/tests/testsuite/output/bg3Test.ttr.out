Just set v = [0] and v = Len = 1 Elements: {0}
In doStuff(), just set x = 2 and x = 2
Just set returned from doStuff(v) and v[0] = 0

